apps/frontend/src/pages/auth/libs/components/forgot-password-form/forgot-password-form.tsx

```diff
@@ -37,7 +39,11 @@ const ForgotPasswordForm: React.FC<Properties> = ({ onSubmit }: Properties) => {
 					type="email"
 				/>
 
-				<Button label="SEND LINK" type="submit" />
+				<Button
+					isDisabled={email === ""}
```

Comment:
```suggestion
					isDisabled={Object.keys(errors).length > 0}
```
But 0 should be constant. Take a look at submit button on sign in, sign up forms

Reply:
this makes the button enabled at the beginning, when the input is empty, this is not good because in requirements I have this:

The "Reset Password" button is disabled when the "Email" field is empty to prevent submission without an email address.